### PR TITLE
FEATURE: Adjust to major overhaul of Flowpack.JobQueue.Common package

### PR DESCRIPTION
**Pull request by bwaidelich**
Adjusts the Beanstalkd Queue implementation to the changes introduced with https://github.com/Flowpack/jobqueue-common/pull/10

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+## Specific options
+
+The `BeanstalkdQueue` supports following options:
+
+| Option                  | Type    | Default                                  | Description                              |
+| ----------------------- |---------| ----------------------------------------:| ---------------------------------------- |
+| defaultTimeout          | integer | 60                                       | Number of seconds new messages are waited for before a timeout occurs (This is overridden by a "timeout" argument in the `waitAndTake()` and `waitAndReserve()` methods |
+| client                  | array   | ['host' => '127.0.0.1', 'port' => 11300] | Beanstalkd connection settings |
+
+*NOTE:* The `DoctrineQueue` currently supports `MySQL`, `PostgreSQL` and `SQLite` backends. You can specify the backend via the `backendOptions`. If you omit this setting, the *current connection* will be re-used (i.e. the currently active Flow database).
```
Comment (kdambekalns) [inline README.md]: What? This is about the `BeanstalkdQueue`…

File: README.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+## Usage
+
+Install the package using composer:
+
+```
+composer require flowpack/jobqueue-beanstalkd
+```
+
+If not already installed, that fetch its requirements, namely `jobqueue-common` and `pda/pheanstalk`.
```
Comment (kdambekalns) [inline README.md]: "that will fetch" I guess…

**Comment by kdambekalns** [comment]
Looks good (apart from two tiny things in the docs). I like the CoC addition, hate `@inheritdoc`. Can't say if it runs, though. :)

File: README.md (hunk 1 above)
Comment (bwaidelich) [inline README.md]: I knew that I wouldn't catch all of these copy/paste gremlins. thanks ;)

**Pull request merged**